### PR TITLE
Refine symptoms flow

### LINKED ISSUE #190: Refine symptoms flow
Now that the navigation flow works, please go through all the report screens and complete anything that's not working or displaying incorrectly. If it's not minor and doesn't have an issue already, you can create one. @Co-Epi/ios

### PR DESCRIPTION
**Pull request by astewart43-gtri**
fixes some label formatting. 
closes #190

### CODE REVIEW

File: CoEpi/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -120,7 +120,7 @@
 "UX.cough.titleWet" = "Wet";
 "UX.cough.subtitleWet" = "\nIt feels like there is mucus \nin my throat and/or chest";
 "UX.cough.titleDry" = "Dry";
-"UX.cough.subtitleDry" = "\nMy throat feels 'tickly' or 'scratchy' without mucus";
+"UX.cough.subtitleDry" = "\nMy throat feels 'tickly' or \n'scratchy' without mucus";
```
Comment (i-schuetz) [inline CoEpi/en.lproj/Localizable.strings:123]: Please ensure that the PRs contain only the changes which are meant to be addressed. This is the only line that should have changed. Use an app like `Sourcetree` or Xcode's git diff to clean up the commits.

Comment (astewart43-gtri) [reply CoEpi/en.lproj/Localizable.strings:123]: My apologies. I noticed that in this and another PR after committing. What is the best way to partially revert a commit? should I simply remove the extra files and commit again?

Comment (astewart43-gtri) [reply CoEpi/en.lproj/Localizable.strings:123]: files have been removed

Comment (i-schuetz) [reply CoEpi/en.lproj/Localizable.strings:123]: Thanks. Next time you also can use e.g. [git reset](https://dev.to/konrad_126/git-reset---explain-49bl) in the command line

**Comment by i-schuetz** [comment]
@astewart43-gtri This change closing #190 means that everything else in the report flow is working / displaying correctly or reported already?

**Comment by astewart43-gtri** [comment]
actually, I noticed that I deleted a file by accident. that has been reverted.
Now it is correct. 
One small issue that I just noticed, the cough type page may have a visual error in Figma. Currently we have it implemented as shown in Figma, but it may be incorrect.
image

we are only able to select one of these options in the current design. should "Select all that apply" to "select one option"

**Comment by i-schuetz** [comment]
Ok, can you bring that up in the ux channel?

**Pull request merged**